survive-the-monsters.js

```diff
@@ -30,9 +30,11 @@ const scenes = {
 scenes.main.preload = function() {
     this.load.image("player", "sprites/players/mvp_player/idle_0.png");
```

Comment:
**issue**: Ensure that the `workaroundSpawnGlitch` function does not introduce any new bugs or side effects. Consider adding tests to verify its behavior.

Reply:
this is only 1 frame of sprite visible at top left corner. workaroud works by eye